components/post_view/reaction/reaction.jsx

```diff
@@ -247,14 +247,19 @@ export default class Reaction extends React.PureComponent {
                 onEnter={this.loadMissingProfiles}
             >
                 <div
+                    id={`post-reaction-${this.props.post.id}-${this.props.emojiName}`}
                     className={className}
                     onClick={handleClick}
                 >
                     <span
+                        id={`${this.props.post.id}-${this.props.emojiName}-emoji`}
                         className='post-reaction__emoji emoticon'
                         style={{backgroundImage: 'url(' + this.props.emojiImageUrl + ')'}}
                     />
-                    <span className='post-reaction__count'>
+                    <span
+                        id={`${this.props.post.id}-${this.props.emojiName}-emoji_count`}
```

Comment:
Same here,
```suggestion
                        id='count'
```
Then get it like ``cy.get(`#postReaction-${postId}-slightly_frowning_face > #count`)``